recipes/newrelic/infrastructure/amazonlinux2.yml

```diff
@@ -111,6 +112,17 @@ install:
           sudo echo 'status_server_enabled: true' >> /etc/newrelic-infra.yml
           sudo echo 'status_server_port: 18003' >> /etc/newrelic-infra.yml
 
+    setup_proxy:
```

Comment:
Look the way it's been done so far in https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/amazonlinux2.yml#L93-L112
Typically we remove any previous value, then add the new value.
One thing to consider is re-deploy. When redeploying we need to take out the value from the config, hence it's easier to remove it all the time and just append the new value if needed to be

Reply:
Explain this more?  Why do we need to take out the value from the config?  In which specific situation does this solution break down?

Reply:
when the next run doesnt use proxy...

Reply:
So installing without a proxy after already choosing to install with a proxy... I think I understand.